bridge-fungible-token/src/utils.sw

```diff
@@ -248,7 +283,70 @@ fn bn_mult(bn: U256, factor: u64) -> (U256, u64) {
 
         result: (U256, u64)
     };
-
     enable_panic_on_overflow();
     result
 }
+
+// TODO: [std-lib] replace when added as a method to U128/U256
+fn bn_div(bn: U256, d: u32) -> (U256, u32) {
+    let m: u64 = 4294967295;
+    let result = (U256::new(), 0u32);
+    asm(bn: __addr_of(bn), d: d, m: m, r0, r1, r2, r3, v0, v1, sum_1, sum_2, q, result: __addr_of(result)) {
+		// The upper 64bits can just be divided normal
+        lw v0 bn i0;
+        mod r0 v0 d; // record the remainder
+        div q v0 d;
+        sw result q i0;
+
+		// The next 64bits are broken into 2 32bit numbers
```

Comment:
Does the formatter need to be run? Some of the spacing seems off

Reply:
Weird. It only looks like that on GH; locally it was formatted nicely. Will reformat it to be sure.

Reply:
Maybe your IDE is rendering tabs as 2 spaces and GitHub as 4 spaces?